src/shogun/evaluation/CrossValidation.cpp

```diff
@@ -280,16 +286,22 @@ float64_t CCrossValidation::evaluate_one_run()
 			/* set feature subset for training */
 			SGVector<index_t> inverse_subset_indices=
 					m_splitting_strategy->generate_subset_inverse(i);
-			m_features->add_subset(inverse_subset_indices);
+
+			CFeatures* features=CFeatures::create_shallow_copy(m_features);
```

Comment:
Yes I had tried to do this initially but no there is no `get_features` or `set_features` in `CMachine`